### DISCUSSION TITLE
Clone is not implemented for `sea_orm::DatabaseConnection` in rocket_example

### DISCUSSION THREAD

**Discussion by Rudi3** [SeaQL/sea-orm#449] [Q&A] [answered]
I tried the `rocket_example` and I'm getting an error in `pool.rs`.

https://github.com/SeaQL/sea-orm/blob/82bad4a376efb875ac32ab601ac2cd9cce191e3f/examples/rocket_example/src/pool.rs#L12

```
the trait bound `sea_orm::DatabaseConnection: std::clone::Clone` is not satisfied

the trait `std::clone::Clone` is not implemented for `sea_orm::DatabaseConnection`rustc(E0277)
pool.rs(12, 5): the trait `std::clone::Clone` is not implemented for `sea_orm::DatabaseConnection`
```

Removing `#[derive(Clone)]` from `SeaOrmPool` fixes the error, but is it intended to have the `Clone` trait?

Features I was using:
```
sea-orm = { version = "0.5.0", features = ["sqlx-postgres", "macros", "runtime-tokio-rustls"]}
sea-orm-rocket = { git = "https://github.com/SeaQL/sea-orm" }
```

**Comment by billy1624** [comment]
Hi @Rudi3, I believe you can safely remove the derive `Clone` here. As it's not necessary. I will create a PR to update the Rocket example in a minute.

https://github.com/SeaQL/sea-orm/blob/82bad4a376efb875ac32ab601ac2cd9cce191e3f/examples/rocket_example/src/pool.rs#L10-L13

**Comment by tyt2y3** [comment, marked as answer]
You missed the `default-features = false`

```toml
sea-orm = { version = "0.5.0", default-features = false, features = ["sqlx-postgres", "macros", "runtime-tokio-rustls"]}
```

**Reply by alper** [reply]
I mean it seems I also missed it, but how did I miss it? Was it written down somewhere and should I have caught it?

**Reply by tyt2y3** [reply]
I explained the situation in https://github.com/SeaQL/sea-orm/issues/438
